translator/config/schema.json

```diff
@@ -468,6 +468,11 @@
                 "minLength": 1,
                 "maxLength": 255
               }
+            },
+            "append_dimensions": {
```

Comment:
Do we wanna add or update some unit tests for this?

Reply:
Was this addressed?

Reply:
Yes I've added append_dimension section to the current unit test for ethtool schema validation.

Reply:
can you update `TestFullConfig` in `ethtool_test.go` so we assert on the changes?

Reply:
done